ui/app/templates/workspace/projects/project/app/release.hbs

```diff
@@ -20,25 +20,19 @@
   </div>
 </PageHeader>
 
-<div class="status-row">
-  <div class="item">
-    {{#if @model.statusReport}}
-      <StatusReportIndicator @statusReport={{@model.statusReport}} />
-    {{/if}}
-  </div>
-
+<StatusReportBar @model={{@model}} @artifactType="Release">
   {{!-- todo(pearkes): API doesn't return this object but just a string of it, so wait for that to be fixed --}}
   {{#if @model.deployment }}
-  <div class="item">
-    <small>
-      Released deployment <LinkTo @route="workspace.projects.project.app.deployment"
-        @models={{array @model.deployment.id}}>
-        <b class="badge badge--version">v{{@model.deployment.sequence}}</b>
-        <code>{{@model.deployment.id}}</code>
-      </LinkTo>
-    </small>
-  </div>
+    <div class="item">
```

Comment:
is this actually displayed on the release view? I see no `{{yield}}` tag in the component's template?